frame/support/procedural/src/construct_runtime/expand/config.rs

```diff
@@ -78,8 +78,8 @@ pub fn expand_outer_config(
 			#fields
 		}
 
-		// GenesisConfig is in the process of being deprecated [https://github.com/paritytech/substrate/issues/14065]
 		#[cfg(any(feature = "std", test))]
+		#[deprecated(note = "Use `RuntimeGenesisConfig` instead.")]
```

Comment:
We may not need the `std` gate since https://github.com/paritytech/substrate/pull/14108 enabled `GenesisConfig` for no-std.